cli/cmdlineparser.cpp

```diff
@@ -577,10 +576,11 @@ bool CmdLineParser::parseFromArgs(int argc, const char* const argv[])
                 else if (!endsWith(mSettings->plistOutput,'/'))
                     mSettings->plistOutput += '/';
 
-                if ((mSettings->plistOutput != "./") && (!Path::folderExist(mSettings->plistOutput)))
+                std::string plistOutput = Path::toNativeSeparators(mSettings->plistOutput);
```

Comment:
the scope of this variable can be reduced and it can be const.

Reply:
No it cannot it should be used in the FileLister::isDirectory check. Changed it to const.